cellprofiler/gui/figure.py

```diff
@@ -693,7 +695,12 @@ def on_mouse_move_measure_length(self, event, x0, y0, x1, y1):
                     self.length_arrow = None
             self.figure.canvas.draw()
             self.Refresh()
-        self.status_bar.SetFields(fields)
+
+        # Update the statusbar
+        if len(fields) > 0:
+            self.status_bar.SetFieldsCount(len(fields))
+            for i in range(len(fields)):
```

Comment:
`for field in fields`?

Reply:
changed to `for idx, field in enumerate(fields):` since SetStatusText() still requires an index